archimedes/archimedes/functions/plotting/colors.py

```diff
@@ -0,0 +1,42 @@
+colors = [
+  '#E69F00',
+  '#56B4E9',
+  '#009E73',
+  '#F0E442',
+  '#0072B2',
+  '#D55E00',
+  '#CC79A7',
+  '#666666',
+  '#AD7700',
+  '#1C91D4',
+  '#007756',
+  '#D5C711',
+  '#005685',
+  '#A04700',
+  '#B14380',
+  '#4D4D4D',
+  '#FFBE2D',
+  '#80C7EF',
+  '#00F6B3',
+  '#F4EB71',
+  '#06A5FF',
+  '#FF8320',
+  '#D99BBD',
+  '#8C8C8C',
+  '#FFCB57',
+  '#9AD2F2',
+  '#2CFFC6',
+  '#F6EF8E',
+  '#38B7FF',
+  '#FF9B4D',
+  '#E0AFCA',
+  '#A3A3A3',
+  '#8A5F00',
+  '#1674A9',
+  '#005F45',
+  '#AA9F0D',
+  '#00446B',
+  '#803800',
+  '#8D3666',
+  '#3D3D3D'
+]
```

Comment:
dittoSeqq::dittoColors(reps=100) actually repeats this set 100 times by default in order to accommodate complexity of more than 40 groups.  (total length = 4000 is probably overkill there, but the thought was towards accommodating coloring by TCR/BCR clones!)

Not saying we need to implement the same way, but we definitely need our code to handle 40+ groupings somehow.

Reply:
In python this is just `colors * 100` :)

Reply:
You can do better by using iterators instead of materialized lists!   `itertools.repeat` for instance.  In this way, even if we need 600000000 different colors, we never exhaust the source. :P

https://www.geeksforgeeks.org/python-itertools-repeat/
:nerd_face: